src/main/java/de/bonndan/nivio/input/dto/LandscapeDescription.java

```diff
@@ -291,11 +288,13 @@ public void setLabels(Map<String, String> labels) {
     }
 
     @Override
+    @JsonAnyGetter
     public String getLabel(String key) {
```

Comment:
@JsonAnyGetter shouldn't be here. According to the documentation a Map must be returned.
https://fasterxml.github.io/jackson-annotations/javadoc/2.6/com/fasterxml/jackson/annotation/JsonAnyGetter.html

Reply:
👍 my bad.

Please move it to the `getLabels()` method.